frontend/app/custom-components/dashboard/profile-card.tsx

```diff
@@ -21,9 +23,18 @@ export function ProfileCard({
           <div>
             <CardTitle className="font-semibold text-xl">{title}</CardTitle>
           </div>
-          <Button variant="outline" size="icon" className={"rounded-[13px]"}>
-            <Settings />
-          </Button>
+          <EditProfile
+            profile={useSearchProfile()}
+            trigger={
+              <Button
+                variant="outline"
+                size="icon"
+                className={"rounded-[13px]"}
```

Comment:
```suggestion
                className="rounded-[13px]"
```
Just a general remark: You dont need brackets if you are passing a hardcoded string. I saw it several times in the code. Maybe you can remove where you see it. But its not a blocker.